Allow is_state to accept a list of values (#25031)
home-assistant/home-assistant.github.io: source/_docs/configuration/templating.markdown

- Iterating `states.domain` will yield each state of that domain sorted alphabetically by entity ID.
- `states.sensor.temperature` returns the state object for `sensor.temperature` (avoid when possible, see note below).
- `states('device_tracker.paulus')` will return the state string (not the object) of the given entity, `unknown` if it doesn't exist, `unavailable` if the object exists but is not yet available.

> - `is_state('device_tracker.paulus', 'home')` will test if the given entity is the specified state. Or if the state is a list of states, test if the given entity is any of the states in the list, for example: `is_state('device_tracker.paulus', ['home', 'work'])`.

Reviewer: ```suggestion
- `is_state` compares an entity's state with a specified state or list of states and return `True` or `False`. `is_state('device_tracker.paulus', 'home')` will test if the given entity is the specified state. `is_state('device_tracker.paulus', ['home', 'work'])` will test if the given entity is any of the states in the list.
```

> - `is_state` compares an entity's state with a specified state or list of states and return `True` or `False`. `is_state('device_tracker.paulus', 'home')` will test if the given entity is the specified state. `is_state('device_tracker.paulus', ['home', 'work'])` will test if the given entity is any of the states in the list.

Reviewer: ```suggestion
- `is_state` compares an entity's state with a specified state or list of states and returns `True` or `False`. `is_state('device_tracker.paulus', 'home')` will test if the given entity is the specified state. `is_state('device_tracker.paulus', ['home', 'work'])` will test if the given entity is any of the states in the list.
```
